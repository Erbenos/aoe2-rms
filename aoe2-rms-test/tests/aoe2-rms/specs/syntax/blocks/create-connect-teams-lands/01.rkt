#lang aoe2-rms
; Connect team members with a road

<LAND-GENERATION>
(create-player-lands
 (terrain-type 'DESERT)
 (number-of-tiles 100))

<CONNECTION-GENERATION>
(create-connect-teams-lands
 (replace-terrain 'DESERT 'ROAD)
 (replace-terrain 'GRASS 'ROAD))
