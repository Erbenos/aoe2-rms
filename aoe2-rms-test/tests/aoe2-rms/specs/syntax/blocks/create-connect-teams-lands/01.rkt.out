<LAND_GENERATION>
create_player_lands {
  terrain_type DESERT
  number_of_tiles 100
}
<CONNECTION_GENERATION>
create_connect_teams_lands {
  replace_terrain DESERT ROAD
  replace_terrain GRASS ROAD
}
